perf_test/sparse/KokkosSparse_par_ilut.cpp

```diff
@@ -132,6 +129,9 @@ void run_par_ilut_test(benchmark::State& state, KernelHandle& kh,
 
 #ifdef USE_GINKGO
 ///////////////////////////////////////////////////////////////////////////////
+static constexpr bool IS_GPU =
```

Comment:
Do you want this change are part of this PR?

Reply:
Sure why not?